spec/stac-geoparquet-spec.md

```diff
@@ -0,0 +1,39 @@
+# STAC GeoParquet Specification
+
+## Overview
+
+This document specifies how to map a set of [STAC Items](https://github.com/radiantearth/stac-spec/tree/v1.0.0/item-spec) into
+[GeoParquet](https://geoparquet.org). It is directly inspired by the [STAC GeoParquet](https://github.com/stac-utils/stac-geoparquet)
+library, but aims to provide guidance for anyone putting STAC data into GeoParquet. 
+
+## Guidelines
+
+Generally most all the fields in a STAC Item should be mapped to a row in GeoParquet. We embrace Parquet structures where possible, mapping
+from JSON into nested structures. We do pull the properties to the top level, so that it is easier to query and use them. The names of the
+most of the fields should be the same in STAC and in GeoParquet.
+
+| Field           | GeoParquet Type    | Required | Details                                            |
+| --------------- | ------------------ | ---------|--------------------------------------------------- |
+| type            | String             | Optional | This is just needed for GeoJSON, so it is optional and not recommended to include in GeoParquet |
+| stac_extensions | List of Strings    | Required | This column is required, but can be blank if no STAC extensions were used |
+| id              | String             | Required | Required, should be unique |
+| geometry        | Binary (WKB)       | Required | For GeoParquet 1.0 this must be well-known Binary. |
+| bbox 	          | List of Decimals   | Required | Can be 4 or 6 decimals, so won't be a fixed size list. |
+| properties      | per field          | Required | Each property should use the relevant Parquet type, and be pulled out of the properties object to be a top-level Parquet field |
+| links           | List of structs    | Required | Each struct in the array should have Strings of `href`, `rel` and `type` |
+| assets          | A struct of assets | Required | Each struct has each full asset key and object as a sub-struct, it's a direct mapping from the JSON to Parquet |
+| collection      | String             | Required | The ID of the collection this Item is a part of |
```

Comment:
I was less thinking about collection-less items, but as I was writing it I was questioning the value of including this field in general. Like I think we'll have the collection link, which seems like the more valuable thing. Having a random string that's the name of the collection seems less useful.

Hrm, thinking about collections - would/could we try to have a 'collection' level metadata somewhere? Like in parquet? If it was like geopackage you could have a table of collection level metadata. I guess the main 'pattern' for using those would be that you're collection level metadata would have a link to the GeoParquet, so maybe we don't need to try to include it?

Reply:
I think this came up once in chatting with @vincentsarago. We could put the collection-level JSON in the Parquet metadata, so that we're able to hold a full representation of the collection json + item json. So ideally a single file could be imported fully into pgstac

Reply:
Makes sense. I added a section in b0d9a6a8cdb73cdba61df53c7eda9adb468fbda4.